geoapi/models/imported_file.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy.sql import func
+from sqlalchemy import (
+    Column, Integer, String,
+    ForeignKey, Boolean, DateTime
+)
+from geoapi.db import Base
+
+class ImportedFile(Base):
+    __tablename__ = 'imported_file'
+
+    id = Column(Integer, primary_key=True)
+    project_id = Column(Integer, ForeignKey('projects.id', ondelete="CASCADE"), index=True)
+    system_id = Column(String, nullable=False, index=True)
+    path = Column(String, nullable=False, index=True)
+    # last_updated is the timestamp on the file itself as to when it was last touched
+    last_updated = Column(DateTime(timezone=True), nullable=False)
```

Comment:
should we add a relationship to what feature(s) correspond to this file

Reply:
I actually started doing that but it started to get complicated. I think would have to do a many-to-many table bubt I couldn't figure out a good place to put that logic. Thoughts?